Core/src/EventData/SeedContainer2.cpp

```diff
@@ -8,23 +8,83 @@
 
 #include "Acts/EventData/SeedContainer2.hpp"
 
+#include "Acts/EventData/Types.hpp"
+
 namespace Acts::Experimental {
 
+SeedContainer2::SeedContainer2() noexcept = default;
+
+SeedContainer2::SeedContainer2(const SeedContainer2 &other) noexcept
+    : m_size(other.m_size) {
+  knownColumns() = other.knownColumns();
+}
+
+SeedContainer2::SeedContainer2(SeedContainer2 &&other) noexcept
+    : m_size(other.m_size) {
+  knownColumns() = std::move(other).knownColumns();
+
+  other.m_size = 0;
+}
+
+SeedContainer2 &SeedContainer2::operator=(
+    const SeedContainer2 &other) noexcept {
+  if (this == &other) {
+    return *this;
+  }
+
+  m_size = other.m_size;
+  knownColumns() = other.knownColumns();
+
+  return *this;
+}
+
+SeedContainer2 &SeedContainer2::operator=(SeedContainer2 &&other) noexcept {
+  if (this == &other) {
+    return *this;
+  }
+
+  m_size = other.m_size;
+  knownColumns() = std::move(other).knownColumns();
```

Comment:
```suggestion
  knownColumns() = std::move(other.knownColumns()(;
```
?

Reply:
`std::move(other).knownColumns()` forces the compiler to use the `&&` overload of that function. `std::move(other.knownColumns())` didn't cut it

Reply:
yeah exactly - this seemed handy to implement move construction and move assignment without duplicating the code